test/schematic_test.exs

```diff
@@ -54,6 +54,22 @@ defmodule SchematicTest do
       assert {:ok, input} == unify(schematic, input)
     end
 
+    test "float/0" do
```

Comment:
can you add floats to the property generators?

should be in these three spots

- https://github.com/mhanberg/schematic/blob/main/test/support/generators.ex#L171
- https://github.com/mhanberg/schematic/blob/main/test/support/generators.ex#L193
- https://github.com/mhanberg/schematic/blob/main/test/support/generators.ex#L227